io.catenax.specialCharacteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,63 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.specialCharacteristics.measurement:1.0.0#> .
+@prefix bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:Measurement a samm:Aspect ;
+   samm:preferredName "SpecialCharacteristicMeasurment"@en ;
+   samm:description "Contains the Measurment for a special characteristic"@en ;
+   samm:properties ( serialPart:localIdentifiers bpn:bpnlProperty :customerPartId :characteristicId :results [ samm:property :measurementType; samm:optional true ] :revisionIndex ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:customerPartId a samm:Property ;
+   samm:preferredName "customerPartId"@en ;
+   samm:description "This number is the identification of the Part"@en ;
+   samm:characteristic :CharacteristicCustomerPartID ;
+   samm:exampleValue "BPN0000000" .
+
+:characteristicId a samm:Property ;
+   samm:preferredName "characteristicId"@en ;
+   samm:description "Identification of the special characteristic in in relation of the specified part"@en ;
+   samm:characteristic :CharacteristicCustomerPartID ;
+   samm:exampleValue "S1" .
+
+:results a samm:Property ;
+   samm:preferredName "measurement_results"@en ;
+   samm:description "List of measurementObjects"@en ;
+   samm:characteristic :ResultList .
+
+:measurementType a samm:Property ;
+   samm:preferredName "Measurement Type"@en ;
+   samm:description "This optional attribute hints the type of the measurement"@en ;
+   samm:characteristic samm-c:Text .
```

Comment:
Does it make sense to define this property as enumeration with the types, that will be defined in the KIT?

![image](https://github.com/user-attachments/assets/98f8aca6-03c4-4d4f-8221-2b944b6b88b4)